config/app.php

```diff
@@ -78,6 +78,8 @@
 
     'providers' => [
         App\Providers\AppServiceProvider::class,
+        Illuminate\Translation\TranslationServiceProvider::class,
```

Comment:
I couldn't get this to work, I'll give it another shot otherwise we can remove this.

Reply:
Resolved, see also https://github.com/hydephp/develop/pull/735